src/main/java/io/pinecone/clients/Index.java

```diff
@@ -779,6 +780,207 @@ public DescribeIndexStatsResponse describeIndexStats(Struct filter) {
         return blockingStub.describeIndexStats(describeIndexStatsRequest);
     }
 
+    /**
+     * {@inheritDoc}
+     * <p> Example</p>
```

Comment:
super nit: I'd lose the leading space in these `Example:` lines and add a colon, to match the other examples in other docstrings.